src/Agent/NewRelic/Agent/Core/Core.csproj

```diff
@@ -71,6 +71,13 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.0" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Remove="NewRelic.Agent.Core.Timing\**" />
+    <EmbeddedResource Remove="NewRelic.Agent.Core.Timing\**" />
+    <None Remove="NewRelic.Agent.Core.Timing\**" />
+    <Protobuf Remove="NewRelic.Agent.Core.Timing\**" />
+  </ItemGroup>
+
```

Comment:
Is this still needed after the file renames?

Reply:
Fixed